doc_src/cmds/string-pad.rst

```diff
@@ -0,0 +1,52 @@
+string-pad - pad characters before and after string
+===================================================
+
+Synopsis
+--------
+
+.. BEGIN SYNOPSIS
+
+::
+
+    string pad [(-r | --right)] [(-c | --char) CHAR] [(-w | --width) INTEGER] [STRING...]
+
+.. END SYNOPSIS
+
+Description
+-----------
+
+.. BEGIN DESCRIPTION
+
+``string pad`` pads specified character before and after the string for each STRING.
+
+The default behaviour is left padding with spaces and default width is the length of string (hence, no padding).
+
+If ``-r`` or ``--right`` is given, only pad after string.
+
+The ``-c`` or ``--char`` switch causes the characters in CHAR to be padded instead of default whitespace character.
```

Comment:
`characters` is slightly confusing, it's only one character

```suggestion
The ``-c`` or ``--char`` switch causes padding with the character CHAR instead of default whitespace character.
```